packages/core/src/entities/schemas/exchangeFilter.js

```diff
@@ -0,0 +1,11 @@
+import { schema } from 'normalizr';
+
+const exchangeFiltersSchema = new schema.Entity(
+  'exchangeFilters',
+  {},
+  {
+    idAttribute: value => value.exchangeFilterItems[0].orderItemUuid,
```

Comment:
In the case of being multiple orderItems being exchanged won't this create a problem since we will only identify the exchangeFilter by the first exchangeFilterItem orderItemUuid?

Reply:
No, because you can't fetch multiple filters at the same time for more than 1 item. Although the `exchangeFilterItems` is an array, It can only have 1 value at a time.

Reply:
What a marvellous API ❤️, approved